### DISCUSSION TITLE
For ultrawide displays, add a 30% / 40% / 30% third option.

### DISCUSSION THREAD

**Discussion by vincentg37** [rxhanson/Rectangle#1534] [General]
To maximize centered window on an ultrawide display, consider allowing the option to adjust the ratio of thirds, such as 30/40/30%, for example.

**Comment by rxhanson** [comment]
Rectangle is intentionally slim to keep it more simple and approachable. I don't have plans to add these in myself, but I would consider a good pull request that adds them in as terminal command configurations (not in the UI).

As an alternative, these can easily be configured in [Rectangle Pro](https://rectangleapp.com/pro) (paid) in the custom size/position tab:
![image](https://github.com/user-attachments/assets/5da0c8e4-f8ac-49e0-9af5-ee08810f075a)

**Reply by vincentg37** [reply]
thank for your reply
